src/main/resources/schema/barnetrygd-feed-schema.json

```
@@ -0,0 +1,119 @@
+{
+  "$id": "http://no.nav.familie.baks.infotrygd.feed",
```

Tenker at de kan ha 2 forskjellige ider
```suggestion
  "$id": ""http://no.nav.familie.ba.infotrygd.feed"",
```